_config.yml

```diff
@@ -95,6 +95,7 @@ JB :
       colorscheme: light
    
   # Settings for analytics helper
+  # Set 'site.safe' to true to enable analytics plugin
```

Comment:
@themez, crap. I can't edit inline. I will have to resubmit this using my account, or you can make the tweak on your end and I'll merge. Lemme know.